src/libcore/char/methods.rs

```diff
@@ -9,6 +9,246 @@ use super::*;
 
 #[lang = "char"]
 impl char {
+    /// The highest valid code point a `char` can have.
+    ///
+    /// A [`char`] is a [Unicode Scalar Value], which means that it is a [Code
+    /// Point], but only ones within a certain range. `MAX` is the highest valid
+    /// code point that's a valid [Unicode Scalar Value].
+    ///
+    /// [`char`]: ../../std/primitive.char.html
+    /// [Unicode Scalar Value]: http://www.unicode.org/glossary/#unicode_scalar_value
+    /// [Code Point]: http://www.unicode.org/glossary/#code_point
+    #[unstable(feature = "assoc_char_consts", reason = "recently added", issue = "71763")]
```

Comment:
Just make it stable.
We need lib team sign off anyway

Reply:
I wouldn't change this to stable yet, I think it makes sense to avoid stabilizing immediately, even if I expect we'll do so relatively quickly.